### DISCUSSION TITLE
Bandwidth usage

### DISCUSSION THREAD

**Discussion by epthewizard** [itsToggle/plex_debrid#552] [Q&A]
I recently set this up and it works great. Thanks for all the hard work. 

It took me a couple tries to get everything to work smoothly how I wanted and I know Plex does transcoding when new files are added to the library. My question is 2 days after I set up rclone on my server I got a message from my ISP saying I had ran through an unusually large amount of data. Like 300GB in 2 days large. I have no idea how this is possible as I thought rclone just mounted the torrents through debrid and nothing was actually downloaded?

The only thing I can think of is re-adding my library to plex a couple times and going through the transcoding but I thought that was more of a cpu/gpu intensive process. If anyone has recommendations please let me know. Thanks.

**Comment by BenJackGill** [comment]
Did you ever figure this out? I'm wondering how much bandwidth to get for a VPS

**Reply by epthewizard** [reply]
Unfortunately no. Plex was way too much for my personal needs so I just got rid of the whole thing and host a stremio server now.
